src/pages/Forum/components/TopicListItem/useTopicListItem.ts

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useState } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { selectIsAuth } from '@redux/slices/settingsSlice';
+
+import { useAddCommentMutation } from '@redux/api/forumApi';
+
+export const useTopicListItem = (numberComments: number) => {
```

Comment:
🧐
Так ближе к правде
```suggestion
export const useTopicListItem = (commentsCount: number) => {
```